src/smc-hub/postgres-server-queries.coffee

```diff
@@ -2035,11 +2036,15 @@ class exports.PostgreSQL extends PostgreSQL
                         stats_tasks.push((cb) => @_count_timespan(table:'projects', field: 'created', age_m: R[tkey], cb: (err, x) => stats.projects_created[K[tkey]] = x; cb(err)))
                         stats_tasks.push((cb) => @_count_timespan(table:'accounts', field: 'created', age_m: R[tkey], cb: (err, x) => stats.accounts_created[K[tkey]] = x; cb(err)))
 
-                async.parallelLimit(stats_tasks, MAP_LIMIT, (err) =>
+                # this was running in parallel, but there is no hurry updating the stats...
```

Comment:
Also, since there is no connection pooling right now, it actually did not run in parallel.